tests/unit/javascript/conftest.py

```diff
@@ -109,8 +109,8 @@ def run_file(self, path: str, expected=None) -> None:
             path: The path to the JS file, relative to the qutebrowser package.
             expected: The value expected return from the javascript execution
         """
-        base_path = os.path.dirname(os.path.abspath(qutebrowser.__file__))
-        with open(os.path.join(base_path, path), 'r', encoding='utf-8') as f:
+        base_path = pathlib.Path(qutebrowser.__file__).resolve().parent
+        with (base_path / path).open('r', encoding='utf-8') as f:
             source = f.read()
```

Comment:
Didn't catch this one before, but I think it can be further simplified using [read_text](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text).